toxiproxy.gemspec

```diff
@@ -10,11 +10,14 @@ Gem::Specification.new do |spec|
   spec.homepage = "https://github.com/Shopify/toxiproxy"
   spec.license = "MIT"
 
+  spec.required_ruby_version = '2.0.0'
```

Comment:
We need `webmock` as a dependency to run the test suite. Is there a way we can set the `required_ruby_version` for development only? I couldn't find one.